db/migrate/20160412085547_create_custom_attribute_to_tag_mappings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCustomAttributeToTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :custom_attribute_to_tag_mappings do |t|
+      t.string :resource_type
+      t.string :name
+      t.text :value
+      t.bigint :category_tag_id
+      t.bigint :tag_id
```

Comment:
Will copy your comment and respond on #7957 which is just the schema change.
(Sorry I haven't updated this PR, I'm almost done rebasing it but something is still broken...)